mmv1/templates/terraform/custom_flatten/secret_version_access.go.tmpl

```diff
@@ -19,7 +19,12 @@ func flatten{{$.GetPrefix}}{{$.TitlelizeProperty}}(v interface{}, d *schema.Reso
 
 	// if this secret version is disabled, the api will return an error, as the value cannot be accessed, return what we have
 	if d.Get("enabled").(bool) == false {
-		transformed["secret_data"] = d.Get("secret_data")
+		if _, ok := d.GetOk("secret_data_wo"); ok {
```

Comment:
is this needed in this PR?

Reply:
reverted here https://github.com/GoogleCloudPlatform/magic-modules/pull/12714/commits/a77b835dadf9b5edd0ac43399e70020165a9ef19